tokens.xml

```diff
@@ -11429,6 +11554,18 @@ Whenever a creature deals combat damage to you, its controller becomes the monar
             <token>1</token>
             <tablerow>1</tablerow>
         </card>
+        <card>
+            <name>Acorn Stash</name>
```

Comment:
I think because of the similarity to experience counters I think it should be put next to those, with the type Counter

Reply:
I'm also not sure on the way to order these "extra" tokens, if you have ideas on unnecessarily rigidifying the structure of the tokens.xml leave a comment on my pr #165

Reply:
I moved it to be alphabetical with the Counters and adjusted it's type and maintype accordingly.